### DISCUSSION TITLE
How can I write migrations with JavaScript?

### DISCUSSION THREAD

**Discussion by gerzhan** [pocketbase/pocketbase#2770] [Q&A] [answered]
Do You have any examples how make migrations with JavaScript?
I am frontend developer and I do not use Golang. I want to use this announced functionality in my project.
I need examples for create(delate) tables(collections) and make(update) views.

**Comment by ganigeorgiev** [comment, marked as answer]
This discussion may help https://github.com/pocketbase/pocketbase/discussions/1464.

There is no documentation yet for the JavaScript migrations.
It will be available with the next release together with the JS app hooks support (you can read more in https://github.com/pocketbase/pocketbase/discussions/2767#discussioncomment-6245393).

Note that you may not need to write custom migrations as the automigrate option is enabled by default and when you make collection changes from the Admin UI it will generate a migration file for you in `pb_migrations` (this folder is safe to be committed in your git repo).
